gateway/src/main/java/org/georchestra/gateway/app/GeorchestraGatewayApplication.java

```diff
@@ -126,6 +127,11 @@ public String loginPage(@RequestParam Map<String, String> allRequestParams, Mode
         return "login";
     }
 
+    @GetMapping(path = "/error/{code}")
+    public String handleError(@PathVariable String code) {
+        return "error/" + code;
+    }
+
```

Comment:
is this necessary ? Relying on the doc provided by @edevosc2c (https://github.com/georchestra/georchestra-gateway/blob/main/docs/custom-error-pages.adoc), as long as we have a template for each http error code under templates/error/xyz.html, it should work.

Reply:
Probably related to https://github.com/georchestra/docker/pull/275/files#diff-2286cdeca85adb587a9640aa34352ba7c2fb6e49c0129dab05892ff67d0d4c58R78

Reply:
yes, it is related to  "traefik.http.middlewares.static-errors-middleware.errors.query=/error/{status}"
But @pmauduit , it did work for me after just adding files under emplates/error (only 404 page worked!).
You can just check locally if you want.

Reply:
Then, as far as I understand it, it is specific to the fact that the docker composition is making use of traefik ? Then we should consider dropping it, and keep the gateway agnostic to whatever is used as a proxy before it.

Reply:
Good idea for me.

Reply:
OK, but how to handle errors for specific projects in such a case ?

Reply:
For me `static-errors-middleware` in traefik is not needed anymore with the gateway because it's the gateway itself that will send the error pages.